docs/developer/payments/lifecycle.mdx

```diff
@@ -274,6 +278,14 @@ be used to provide the history of `transactionItem` actions.
 The provided `amount` will be used to reduce `transaction.chargedAmount`
 (`transaction.chargedAmount -= amount`).
 
+### Refund events
+
+:::warning
+Refund events should only be triggered if the transaction has been previously charged.
+Triggering a refund for uncharged transactions can lead to negative `chargedAmount`.
```

Comment:
Is it "legal" operation? What happens if we have negative chargeAmount? Does it make sense to refund more money than was charged initially?